packaging/nuget/nservicebus.host.nuspec

```diff
@@ -20,6 +20,7 @@
   </metadata>
   <files>
     <file src="..\..\binaries\NServiceBus.Host.???" target="lib\net452" />
-    <file src="..\..\packaging\nuget\nservicebus.host\*.*" target="tools" />
+    <file src="..\..\packaging\nuget\nservicebus.host\install.ps1" target="tools" />
+    <file src="..\..\packaging\nuget\nservicebus.host\EndpointConfig.cs.pp" target="Content" />
```

Comment:
Only lib had that before, but it makes sense to scope everything the same. Will push up that change.